tests/Unit/DataStructures/Test_Variables.cpp

```diff
@@ -470,15 +491,23 @@ SPECTRE_TEST_CASE("Unit.DataStructures.Variables.PrefixSemantics",
         });
   };
 
-  check_variables(vars_0, variable_type(4, 8.0));
-  check_variables(vars_1, variable_type(4, 6.0));
-
-  prefix_variable_type prefix_vars_2(4, 4.);
-  prefix_variable_type prefix_vars_3(4, 2.);
-  vars_0 = prefix_vars_2;
-  check_variables(vars_0, variable_type(4, 4.0));
-  vars_0 = std::move(prefix_vars_3);
-  check_variables(vars_0, variable_type(4, 2.0));
+  check_variables(vars_to,
```

Comment:
all of these helper check functions should just call a CHECK_VARIABLES_APPROX in `TestingFramework.hpp`. @nilsdeppe @wthrowe is there a good reason this shouldn't be done?